ssl/record/rec_layer_s3.c

```diff
@@ -602,6 +612,15 @@ int ssl3_write_bytes(SSL *s, int type, const void *buf_, size_t len,
             }
         }
 
+#ifndef OPENSSL_NO_TLSEXT
+        if (USE_MAX_FRAGMENT_LENGTH_EXT(s->session)) {
+            unsigned int max_fragment_len = GET_MAX_FRAGMENT_LENGTH(s->session);
+            for (j = 0; j < numpipes; j++) {
+                if (pipelens[j] > max_fragment_len)
+                    pipelens[j] = max_fragment_len;
+            }
+        }
+#endif
```

Comment:
I dig this point, and see what i speak about.
Code is checking that `split_send_fragment <= max_fragment_len` .
I modify this  PR to convert the max_fragment_len code setting into a max_fragment_len value...
so the fragment len is easily limited ...

I noticed that there is no option in `s_client` or `s_server` to set `max_send_fragment`.
I will do a separated PR to fix it.